dso-l2/src/main/java/com/tc/objectserver/impl/ServerPersistenceVersionChecker.java

```diff
@@ -47,24 +47,12 @@ public ServerPersistenceVersionChecker(ClusterStatePersistor clusterStatePersist
     this.versionCompatibility = versionCompatibility;
   }
 
-  private boolean checkVersion(Version persisted, Version current) {
-    if (persisted != null) { return versionCompatibility.isCompatibleServerPersistence(persisted, current); }
-    return true;
-  }
-
   public void checkAndSetVersion() {
```

Comment:
Worth changing the name of this method to reflect what it is doing?

Reply:
I would rather refactor as well, as the method has an implicit side effect on a class field.